src/components/StatsLeaderboard.vue

```diff
@@ -144,13 +147,16 @@ export default {
           const matchesThisWeek = playerMatches[weekNum] || [];
           const wins = matchesThisWeek.filter((match) => match.result === Result.WON);
           res.total += wins.length;
-          res[`${weekNum}`] = wins.length;
+          res[`${weekNum}`] = {
+            wins: wins.length,
+            matchesPlayed: matchesThisWeek.length,
```

Comment:
I believe that matchesThisWeek could have incomplete matches. We should double check this, but we might need to `.filter()` this based on the match result to only include complete matches

Reply:
I stand corrected; there are no incomplete matches in the DTO; this should work as written